samples/boards/sensortile_box_pro/ble-fw-upgrade/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_GPIO=y
+CONFIG_BT=y
```

Comment:
Actually it does not build correctly without it. But, but, this is only because I decided to include the include file from the blob ONLY in case of CONFIG_BT=y. So, now I'm evaluating this part and use ```CONFIG_DT_HAS_ST_HCI_SPI_V2_ENABLED``` instead (actually it should be ```CONFIG_DT_HAS_ST_HCI_SPI_V1_ENABLED``` as well)

Reply:
I guess that this code would be ok to cover the mkbox boards cases, can you confirm it @msmttchr ?

```
if(CONFIG_DT_HAS_ST_HCI_SPI_V1_ENABLED OR CONFIG_DT_HAS_ST_HCI_SPI_V2_ENABLED)
  add_subdirectory(ble_firmware)
endif()
```

See https://github.com/zephyrproject-rtos/hal_stm32/pull/217/ (you may also add your own comments there...)